### ISSUE TITLE
Subscribing to specific events (INSERT,UPDATE,DELETE) doesn't work

### ISSUE THREAD

**Issue by awalias** [supabase/supabase#25]
See [`supabase/supabase/libraries/supabase-js/test/integration/testRealtime.js`](https://github.com/supabase/supabase/blob/499883f75a35e9163de54ae5771b03d1807a84fb/libraries/supabase-js/test/integration/testRealtime.js#L74)

The tests that test for specific eventType are currently failing

**Comment by awalias** [comment]
I also verified that 
```javascript
    const subscription = supabase
      .from('messages')
      .on('*', callbackAction)
      .subscribe()
```
works ok, so the problem is in the EventType and not the table level filtering

**Comment by awalias** [comment]
Can verify that the binding gets set up on the `supabase-js` side:

```javascript
channel: Channel {
  topic: 'realtime:*',
  ... ,
  bindings: [
    { event: 'phx_close', callback: [Function] },
    { event: 'phx_error', callback: [Function] },
    { event: 'phx_reply', callback: [Function] },
    { event: 'INSERT', callback: [Function] }
  ],
}
```

I tried looking on the realtime side @kiwicopple but my elixer not good enough to confirm that the messages get send on the channel with the correct event name `INSERT`

**Comment by kiwicopple** [comment]
Can confirm it's missing functionality in the elixir server:
https://github.com/supabase/realtime/blob/9a6f3d97840a9789d01b4bf24303373bfee95185/server/lib/realtime_web/channels/realtime_channel.ex#L29

This should parse out the eventType and then broadcast. eg:

```
RealtimeWeb.Endpoint.broadcast_from!(self(), topic, "*", txn)
RealtimeWeb.Endpoint.broadcast_from!(self(), topic, eventType, txn)
```
Will fix today.

**Comment by kiwicopple** [comment]
Server fixed on 0.7.0 https://github.com/supabase/realtime/issues/20

**Comment by kiwicopple** [comment]
Hey @awalias was this one solved now on your side?

**Comment by awalias** [comment]
I think it is fixed, but will just need https://github.com/supabase/realtime/issues/21 in order to confirm (since it causes the integration tests to fail)

**Comment by awalias** [comment]
last thing remaining is `04:33:14.166 [error] GenServer #PID<0.2513.0> terminating
** (KeyError) key :record not found in: %Realtime.Adapters.Changes.DeletedRecord{columns: [%Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "id", type: "int8", type_modifier: 4294967295}, %Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "inserted_at", type: "timestamp", type_modifier: 4294967295}, %Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "updated_at", type: "timestamp", type_modifier: 4294967295}, %Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "data", type: "jsonb", type_modifier: 4294967295}, %Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "message", type: "text", type_modifier: 4294967295}, %Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "user_id", type: "int8", type_modifier: 4294967295}, %Realtime.Decoder.Messages.Relation.Column{flags: [:key], name: "channel_id", type: "int8", type_modifier: 4294967295}], commit_timestamp: ~U[2020-03-16 04:33:14Z], old_record: %{"channel_id" => "1", "data" => nil, "id" => "50", "inserted_at" => "2020-03-16 04:33:14.144909", "message" => "delete test", "updated_at" => "2020-03-16 04:33:14.144909", "user_id" => "1"}, schema: "public", table: "messages", type: "DELETE"}`

**Comment by awalias** [comment]
fixed in `realtime:0.7.2` and `supabase-dev:0.1.8`
